components/Input/Input.css

```diff
@@ -0,0 +1,98 @@
+@import url(../../style.css);
+
+.input,
+.text-area {
+  background-color: var(--ternary-color);
+  border: 1px solid var(--primary-dark);
+  outline: none;
+  color: var(--secondary-color);
+  padding: 7px;
+  position: relative;
+}
+
+.input:hover,
+.input:focus,
+.text-area:hover,
+.text-area:focus {
+  border: 1px solid var(--primary-color);
+  background-color: var(--ternary-dark);
+}
+
+.input-validation-msg {
+  content: "wrong password!";
+  color: red;
+  display: block;
+  position: absolute;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+  appearance: none;
+
+  width: 1.15em;
+  height: 1.15em;
+  border: 0.15em solid var(--ternary-dark);
+
+  display: grid;
+  place-content: center;
+}
+
+input[type="radio"]::before,
+input[type="checkbox"]::before {
+  content: "";
+  width: 0.65em;
+  height: 0.65em;
+
+  transform: scale(0);
+  /* transition: 120ms transform ease-in-out; */
+
+  background-color: var(--primary-color);
+}
+
+input[type="radio"]::before {
+  border-radius: 50%;
+}
+
+input[type="radio"]:checked::before,
+input[type="checkbox"]:checked::before {
+  transform: scale(1);
+}
+
+.form-example {
+  height: 20rem;
+  width: 20rem;
+  display: flex;
+  flex-direction: column;
+  padding: 3rem;
+  background-color: var(--ternary-color);
+  border: 1px solid var(--primary-color);
+}
+.form-error-msg {
+  color: red;
+  display: none;
+}
+
+.input:invalid:focus {
+  border: 2px solid red;
+  /* background-color: rgb(70, 5, 5); */
+}
```

Comment:
You can use CSS variables instead of directly giving the color name.

Rest all looks good!